presto-main/src/main/java/io/prestosql/operator/scalar/JoniRegexpFunctions.java

```diff
@@ -307,4 +307,97 @@ private static void validateGroup(long group, Region region)
             throw new PrestoException(INVALID_FUNCTION_ARGUMENT, format("Pattern has %d groups. Cannot access group %d", region.numRegs - 1, group));
         }
     }
+
+    private static void validPattern(JoniRegexp pattern)
```

Comment:
this wasn't validated for other regexp functions. I think we should support empty pattern instead. For example pattern might be another column and we don't want query to fail because of this.

Reply:
Thanks @sopel39  I have no idea what number should be returned if 'pattern' is empty, any idea?

Reply:
Every position matches empty pattern, see how `regex_replace` works.